### PR TITLE
jquery: Add Black Lives Matter support banner

### PR DESCRIPTION
**Pull request by jorydotcom**
per slack discussion; this is global across all jquery projects cc @timmywil

### CODE REVIEW

File: themes/jquery/header.php [hunk 1]
```diff
@@ -104,6 +104,11 @@
 	</section>
 </header>
 
+<div id="banner-secondary">
+    <h2>#BlackLivesMatter</h2>
+	<p>To Donate, see <a href="(https://www.reclaimtheblock.org/)">Reclaim the Block</a>'s <a href="https://bit.ly/fundthecommunity">list of organizations to support</a>
```
Comment (timmywil) [inline themes/jquery/header.php:109]: Minor spacing nit.

File: themes/jquery/header.php [hunk 2]
```diff
@@ -104,6 +104,11 @@
 	</section>
 </header>
 
+<div id="banner-secondary">
```
Comment (mgol) [inline themes/jquery/header.php:107]: This is an existing ID since there's already some CSS for it. jqueryui.com relies on it, for example. Modifying styles for that one may be tricky. It might be safer to create a new id or a class and style it from scratch.

File: themes/jquery/header.php [hunk 3]
```diff
@@ -64,7 +64,7 @@
 					<li><a href="https://plugins.jquery.com/">Plugins</a></li>
 					<li class="dropdown"><a href="https://contribute.jquery.org/">Contribute</a>
 						<ul>
-							<li><a href="https://js.foundation/CLA">CLA</a></li>
+							<li><a href="https://cla.js.foundation/jquery/">CLA</a></li>
```
Comment (mgol) [inline themes/jquery/header.php:67]: Can we do this in a separate PR? :)

File: themes/jquery/header.php (hunk 2 above)
Comment (jorydotcom) [reply themes/jquery/header.php:107]: OK to add a new class to base.css, perhaps in the nav section?

File: themes/jquery/header.php (hunk 3 above)
Comment (jorydotcom) [reply themes/jquery/header.php:67]: of course, I'll do a sep. pr for all the link updates....

File: themes/jquery/header.php [hunk 4]
```diff
@@ -104,6 +104,11 @@
 	</section>
 </header>
 
+<div id="banner-blm">
+    <h2>#BlackLivesMatter</h2>
```
Comment (mgol) [inline themes/jquery/header.php:108]: Please use tabs instead of spaces here. :)

**Comment by timmywil** [comment]
Before I merge, I'd like to make sure this looks okay on mobile. A screenshot would suffice. I'd check it myself, but I don't want to get sites running locally ;)

File: themes/jquery/header.php (hunk 4 above)
Comment (timmywil) [reply themes/jquery/header.php:108]: Maybe we could add a `.editorconfig` to this repo. Spaces are my default as well.

Comment (timmywil) [reply themes/jquery/header.php:108]: Not suggesting that for this PR, just a thought.

Comment (jorydotcom) [reply themes/jquery/header.php]: should be fixed now!

**Comment by jorydotcom** [comment]
Screen Shot 2020-07-01 at 5 13 19 PM

shared in slack, here for posterity!

File: themes/jquery/header.php (hunk 2 above)
Comment (jorydotcom) [reply themes/jquery/header.php]: added banner-blm to section with other global banners

File: themes/jquery/header.php (hunk 4 above)
Comment (mgol) [reply themes/jquery/header.php]: @timmywil yes, that's a good idea.

**Comment by fnagel** [comment]
@jorydotcom Next time, please make sure everybody involved is informed before adding a political banner.

I got personal emails about this without even knowing this has been done.

**Comment by timmywil** [comment]
That's more on me. Jory simply volunteered to implement the banner. However, we discussed this in public channels, such as the jquery channel on the openjs slack and in the jQuery meetings on gitter. I don't know what kind of emails you're getting, but if someone is bothered by our support of this movement, I would advise you to ignore them.
